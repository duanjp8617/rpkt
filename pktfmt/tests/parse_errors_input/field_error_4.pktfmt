packet FakeProto {
    header = [
        fake_field1 = Field {
            bit = 1, 
            arg = bool,
            default = 5
        }
    ]
}

