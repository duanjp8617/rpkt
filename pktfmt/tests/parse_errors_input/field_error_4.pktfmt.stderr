ast error
at /home/djp/pkt-rs/pktfmt/tests/parse_errors_input/field_error_4.pktfmt 3:23-7:9
                          ~~~~~~~
3 |         fake_field1 = Field {
4 |             bit = 1, 
5 |             arg = bool,
6 |             default = 5
7 |         }
    ^^^^^^^^^
field error 4:
invalid default 5 for boolean arg, should be true or false
